app/routes/helpers.py

```diff
@@ -0,0 +1,14 @@
+from flask import Blueprint, jsonify, make_response, abort, request
+from app.models.planet import Planet
+
+def validate_planet(id):
```

Comment:
In the future, you might need to make 'helper.py' more specific. Think about Task List, you have Goal and Task, you may need helpers for each. If you refactor helper methods into a helper file then you could have goal_helper.py and task_helper.py